users/models.py

```diff
@@ -75,8 +75,13 @@ class User(SafeDeleteModel, AbstractUser):
     status = models.CharField(
         max_length=255, choices=user_status_choices, default="approved"
     )
-    auth_type = models.CharField(max_length=255, choices=auth_type_choices, null=True)
     unique_id = models.CharField(max_length=255, null=True)
+    org = models.ForeignKey(
+        Organization, on_delete=models.DO_NOTHING, 
+        related_name="org",
```

Comment:
why use this field?

Reply:
This is the error I get if I don't use the `related_name` field
```bash
ERRORS:
users.User.org_id: (fields.E304) Reverse accessor for 'User.org_id' clashes with reverse accessor for 'User.organizations'.
	HINT: Add or change a related_name argument to the definition for 'User.org_id' or 'User.organizations'.
users.User.organizations: (fields.E304) Reverse accessor for 'User.organizations' clashes with reverse accessor for 'User.org_id'.
	HINT: Add or change a related_name argument to the definition for 'User.organizations' or 'User.org_id'.
```
I'm guessing this is because we already have a ManyToMany relationship defined with `Organization`. So if we don't give a `related_name` it gets confused between the two.